packages/lab/src/dialog/DialogContent.css

```diff
@@ -1,13 +1,14 @@
 .saltDialogContent {
   color: var(--salt-content-primary-foreground);
-  padding-left: var(--salt-spacing-300);
-  padding-right: var(--salt-spacing-300);
-  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
```

Comment:
I feel like this is too opinionated for something that should be similar to card.